### DISCUSSION TITLE
Support tailwind animate

### DISCUSSION THREAD

**Discussion by nicolasthy** [tw-in-js/twind#478] [Ideas]
Hello 👋 
Sorry if this has already been addressed but I couldn't find anything related. Is there a way to use [tailwind-animate](https://github.com/jamiebuilds/tailwindcss-animate) plugin with Twind?

**Comment by edouardr** [comment]
I am currently working with [shadcn-ui](https://ui.shadcn.com) which uses it. Would be much appreciated to have a preset for it, to have a full support of shadcn-ui.

**Comment by firebotQL** [comment]
Hi. Any news on this one? Don't think it's straightforward on converting tailwind plugins: [require("tailwindcss-animate")], but I need it as well...
